Python/HAT/draw.py

```
@@ -73,3 +160,65 @@ def incidencePlot(H, shadeRows=True, connectNodes=True, dpi=200, edgeColors=None
     plt.xticks([])
     
     return plt.gca()
+
+def rubber_bands(
+    HG,
+    pos=None,
+    with_color=True,
+    with_node_counts=False,
+    with_edge_counts=False,
+    layout=nx.spring_layout,
+    layout_kwargs={},
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)